packages/universal-router-sdk/src/utils/constants.ts

```diff
@@ -1,130 +1,88 @@
 import { ChainId } from '@pancakeswap/chains'
 import { Address } from 'viem'
-import { ChainConfig } from './types'
-
-const WETH_NOT_SUPPORTED_ON_CHAIN = '0x0000000000000000000000000000000000000000'
+import { ChainConfig } from '../entities/types'
 
 const CHAIN_CONFIGS: { [key: number]: ChainConfig } = {
```

Comment:
what about we have two separate two chain based map?
```suggestion
const UNIVERSAL_ROUTER_ADDRESSES: Record<ChainId, Address> = {
```

Reply:
how do u mean can u explain more

Reply:
like this? And I think it's better we put permit2 addresses to permit2-sdk
```typescript
const UNIVERSAL_ROUTER_ADDRESSES: Record<ChainId, Address> = {
  [ChainId.BSC]: '0x',
  ...
}

const PERMIT2_ADDRESSES: Record<ChainId, Address> = {
  [ChainId.BSC]: '0x',
  ...
}
```

Reply:
yup thats for explaining. making changes now